### PR TITLE
Marketplace protocol mapper

### PR DESCRIPTION
**Pull request by JulianaMeyerArruda**
#### Types of changes
- [X] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

Adding the VTEX Mapper endpoints to our Marketplace Protocol API!

### CODE REVIEW

File: VTEX - Marketplace Protocol.json [hunk 1]
```diff
@@ -523,7 +523,18 @@
...
+					},
+                    {
+						"name": "orderId",
+						"in": "path",
+						"description": "Id of the order being fulfilled",
+						"required": true,
+						"style": "simple",
+						"schema": {
+							"type": "string",
+							"default": ""
```
Comment (karenkrieger) [inline VTEX - Marketplace Protocol.json:537]: Adicionar exemplo

File: VTEX - Marketplace Protocol.json [hunk 2]
```diff
@@ -613,7 +624,18 @@
...
+					},
+                    {
+                        "name": "orderId",
+                        "in": "path",
+                        "description": "Id of the order being fulfilled",
+                        "required": true,
+                        "style": "simple",
+                        "schema": {
+                            "type": "string",
+                            "default": ""
```
Comment (karenkrieger) [inline VTEX - Marketplace Protocol.json:636]: Adicionar exemplo

File: VTEX - Marketplace Protocol.json [hunk 3]
```diff
@@ -613,7 +624,18 @@
 							"type": "string",
 							"example": "application/json"
 						}
-					}
+					},
+                    {
+                        "name": "orderId",
+                        "in": "path",
+                        "description": "Id of the order being fulfilled",
```
Comment (karenkrieger) [inline VTEX - Marketplace Protocol.json:631]: `Id` -> `ID`

File: VTEX - Marketplace Protocol.json [hunk 4]
```diff
@@ -523,7 +523,18 @@
...
 						"schema": {
 							"type": "string",
-							"example": "application/json"
+							"default": "application/json"
+						}
+					},
+                    {
+						"name": "orderId",
+						"in": "path",
+						"description": "Id of the order being fulfilled",
```
Comment (karenkrieger) [inline VTEX - Marketplace Protocol.json:532]: `Id` -> `ID`

File: VTEX - Marketplace Protocol.json [hunk 5]
```diff
@@ -1162,6 +1184,121 @@
...
+                  },
+                  {
+                    "name": "id",
+                    "in": "path",
+                    "description": "Code identifying returned in the *VTEX Mapper Registration* endpoint.",
+                    "required": true,
+                    "style": "simple",
+                    "schema": {
+                      "type": "string",
+                      "default": ""
```
Comment (karenkrieger) [inline VTEX - Marketplace Protocol.json:1279]: Adicionar exemplo

**Review by karenkrieger — APPROVED** [review]
Só resolver esses comentários que podemos dar merge!

File: VTEX - Marketplace Protocol.json (hunk 1 above)
Comment (JulianaMeyerArruda) [reply VTEX - Marketplace Protocol.json:537]: Feito!

File: VTEX - Marketplace Protocol.json (hunk 2 above)
Comment (JulianaMeyerArruda) [reply VTEX - Marketplace Protocol.json:636]: Feito!

File: VTEX - Marketplace Protocol.json (hunk 3 above)
Comment (JulianaMeyerArruda) [reply VTEX - Marketplace Protocol.json]: Feito!

File: VTEX - Marketplace Protocol.json (hunk 4 above)
Comment (JulianaMeyerArruda) [reply VTEX - Marketplace Protocol.json]: Feito!

File: VTEX - Marketplace Protocol.json (hunk 5 above)
Comment (JulianaMeyerArruda) [reply VTEX - Marketplace Protocol.json:1279]: Feito!

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**